doc/source/user_guide/ug_implementation.rst

```diff
@@ -213,10 +213,11 @@ ghost cells, and the same numbering system is applied to each of the
 four subdomains.
 
 The user sets the ``NTASKS`` and ``NTHRDS`` settings in **cice.settings** 
-and chooses a block size ``block_size_x`` :math:`\times`\ ``block_size_y``, 
-``max_blocks``, and decomposition information ``distribution_type``, ``processor_shape``, 
-and ``distribution_type`` in **ice_in**. That information is used to
-determine how the blocks are
+and chooses a block size, ``block_size_x`` :math:`\times`\ ``block_size_y``,
+and decomposition information ``distribution_type``, ``processor_shape``, 
+and ``distribution_type`` in **ice_in**. ``max_blocks`` is computed
+automatically if set to a value of -1, but it can also be set by the user.
```

Comment:
I still think this paragraph is out of order:

_The user sets the NTASKS and NTHRDS settings in cice.settings and chooses a block size, block_size_x block_size_y, and decomposition information distribution_type, processor_shape, and distribution_wgt in ice_in. If max_blocks=-1, the model will calculate the number of blocks needed for each task. max_blocks can also be set by the user, although this may use extra memory and the model will abort if max_blocks is set too small for the decomposition. **This information** is used to determine how the blocks are distributed across the processors, and how the processors are distributed across the grid domain._

"This information" should refer to the information in the first sentence but at the moment it reads"This information" refers to "max_blocks"

Reply:
You're right, thanks for checking again.  I have refactored that paragraph a bit.  I think it's better now.  Let me know if you have any concerns.

Reply:
Looks good. Thankyou